net/extras/genInventoryFile.py

```diff
@@ -54,6 +54,9 @@ def writeInventory(self, outFd):
             outFd.write(var_line)
             var_line = "netplugin_if={} ".format(connInfo[node]['data'])
             outFd.write(var_line)
+            role = connInfo[node].get('role', 'worker')
```

Comment:
What we need to do is define two different roles for netplugin_node as master and worker and call them appropriately for each node.